packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -36,6 +36,12 @@ class ConfigManager implements IConfigManager {
    * config related information in global state
    */
   init() {
+    effect(() => {
```

Comment:
can we create a method called attachEffects that contains all similar effect logic o tey are gathered in one place please